api/bucketsd/service.go

```diff
@@ -122,6 +127,20 @@ func (l buckLock) Key() string {
 	return string(l)
 }
 
+// StartRepublishing initializes a key republishing cron
+func (s *Service) StartRepublishing(schedule string, session string) error {
+	s.republisher = cron.New()
```

Comment:
create at time of start so i didn't need to add a NewService method at this time.